pkg/resources/warehouse_acceptance_test.go

```diff
@@ -620,11 +621,11 @@ func TestAcc_Warehouse_Validations(t *testing.T) {
 			},
 			{
 				Config:      warehouseWithAutoResumeConfig(id.Name(), "other"),
-				ExpectError: regexp.MustCompile(`expected auto_resume to be one of \["true" "false"], got other`),
+				ExpectError: regexp.MustCompile(`expected \[\{\{} auto_resume}] to be one of \["true" "false"], got other`),
 			},
 			{
-				Config:      warehouseConfigWithMaxConcurrencyLevel(id.Name(), -1),
-				ExpectError: regexp.MustCompile(`expected max_concurrency_level to be at least \(1\), got -1`),
+				Config:      warehouseConfigWithMaxConcurrencyLevel(id.Name(), -2),
+				ExpectError: regexp.MustCompile(`expected max_concurrency_level to be at least \(1\), got -2`),
```

Comment:
was it conflicting with the -1 default or that's a parameter? 🤔

Reply:
It was not conflicting but because we treat -1 special, I wanted to use other value (it was working with -1).